platform/nodejs/eslintrc.json

```diff
@@ -9,7 +9,20 @@
         "sourceType": "module"
     },
     "rules": {
-        "indent": [ "warn", 4 ],
+        "indent": [
+          "warn",
+          4,
+          {
+            "ArrayExpression": "off",
+            "CallExpression": { "arguments": "first" },
+            "MemberExpression": "off",
+            "ObjectExpression": "off",
+            "ignoreComments": true,
+            "ignoredNodes": [
+              "AssignmentExpression:has(Literal)"
```

Comment:
This addresses the following kind of code:

```js
             this.dotTokenHash = 0x10000000;
             this.anyTokenHash = 0x20000000;
        this.anyHTTPSTokenHash = 0x30000000;
         this.anyHTTPTokenHash = 0x40000000;
              this.noTokenHash = 0x50000000;
           this.emptyTokenHash = 0xF0000000;
```

The string `AssignmentExpression:has(Literal)` is a [selector](https://eslint.org/docs/developer-guide/selectors). It matches a node in the AST. There's an AST explorer here just for playing around: https://astexplorer.net/